app/workers/evss/disability_compensation_form/submit_uploads.rb

```diff
@@ -26,6 +26,7 @@ def perform(submission_id, upload_data)
         upload_data = upload_data.first if upload_data.is_a?(Array) # temporary for transition
         guid = upload_data&.dig('confirmationCode')
         with_tracking("Form526 Upload: #{guid}", submission.saved_claim_id, submission.id) do
+          # [wipn8923] update to use flipper
           sea = SupportingEvidenceAttachment.find_by(guid:)
```

Comment:
This is where the inheritance issue is

Recommend: flipper around this, SupportingEvidenceAttachement is evss path, for everything for LH up object change replicate all in LH namespace, inherit at the top from the LH document uploader base.

Dupe the tests as well from the EVSS equivalent.

Reply:
Took a crack at this in https://github.com/department-of-veterans-affairs/vets-api/pull/13835/commits/2fb7784c80147a0aaa84d647c4f6fbcbeadfa1ee